docs/synphot/filter_par.rst

```diff
@@ -0,0 +1,124 @@
+    >>> bp = SpectralElement.from_file(filename)
+    >>> n_lambda, lambda_0, delta_lambda, tr_max, fft_pars = filter_to_fft(bp)
+    >>> n_lambda  # Number of elements in wavelengths
+    10000
+    >>> lambda_0  # Starting value of wavelengths  # doctest: +FLOAT_CMP
+    <Quantity 3479.999 Angstrom>
+    >>> delta_lambda  # Median wavelength separation  # doctest: +FLOAT_CMP
+    <Quantity 0.66748047 Angstrom>
+    >>> tr_max  # Peak value of throughput  # doctest: +FLOAT_CMP
+    <Quantity 0.241445>
+    >>> fft_pars  # FFT parameters  # doctest: +SKIP
+    [(407.5180314841658+7.494005416219807e-16j),
+     (-78.52240189503877-376.53990235136575j),
+     (-294.86589196496584+127.25464850352665j),
+     (130.20273803287864+190.84263652863257j),
+     (96.62299079012317-91.70087676328245j),
+     (-32.572468348727654-34.227696019221035j),
+     (-8.051741476066471-21.354793540998294j),
+     (-51.708676896903725+6.883836090870033j),
+     (13.08719675518801+54.48177212720124j),
+     (38.635087381362396-13.02803811279449j)]
+
+.. TODO: Only skipping the fft_pars comparison above because output is very
+   different for NUMPY_LT_1_17. Unskip it and replace with +FLOAT_CMP when
+   Numpy minversion is 1.17.
+
+It is up to you to decide how to store this data, though storing it in a
+table format is recommended. In fact, if you have many filters to parameterize,
+:func:`~synphot.filter_parameterization.filters_to_fft_table`
+will store the results in a table for you::
+
+    >>> from synphot.filter_parameterization import filters_to_fft_table
+    >>> mapping = {'HST/ACS/HRC/F555W': (bp, None)}
+    >>> filter_pars_table = filters_to_fft_table(mapping)
+    >>> filter_pars_table  # doctest: +SKIP
+    <Table length=1>
+          filter      n_lambda ...                  fft_9
+                               ...
+          str17        int...  ...                complex128
+    ----------------- -------- ... ---------------------------------------
+    HST/ACS/HRC/F555W    10000 ... (38.635087381362396-13.02803811279449j)
+    >>> filter_pars_table.write('my_filter_pars.fits')  # doctest: +SKIP
+
+.. TODO: Only skipping the filter_pars_table comparison above because output
+   is slightly different for NUMPY_LT_1_17. Unskip it and replace with
+   +FLOAT_CMP +ELLIPSIS when Numpy minversion is 1.17.
+
+.. _filter_fft_construction:
+
+Reconstructing Filter from FFT
+------------------------------
+
+Once you have a parameterized filter (see :ref:`filter_fft_generation`),
+you can reconstruct it for use using
+:func:`~synphot.filter_parameterization.filter_from_fft`.
+Following from the example above::
+
+    >>> from synphot.filter_parameterization import filter_from_fft
+    >>> reconstructed_bp = filter_from_fft(
+    ...     n_lambda, lambda_0, delta_lambda, tr_max, fft_pars)
+
+For this particular example using HST ACS/HRC F555W filter, perhaps 10
+parameters are not quite sufficient. Therefore, caution needs to be exercised
```

Comment:
Maybe there should be one more snippet below that shows a reconstruction with more than 10 fft coefficients?  That would also demonstrate how to actually do that for the user (I know it's in the API, but since it's coming up, might as well show it here).

Reply:
I'll chuck this as follow-up issue because I have no idea what you mean anymore. I haven't thought about this for 2 years. 😆